types/components/Accordion.d.ts

```diff
@@ -0,0 +1,14 @@
+import { GeneratedPropTypes } from '../types';
+import React from 'react';
+export declare type AccordionItemProps = {
+    children: any;
```

Comment:
using the same as above
```suggestion
    children: React.ReactNode;
```